### ISSUE TITLE
The number of blocks mined in a single go should be limited

### ISSUE THREAD

**Issue by echosa** [echosa/mine_it_all#1]
There are two issues here, one worse than the other.
1. The worse issue is that, currently, if you veinmein dirt, it'll keep trying to vein mine it _all_. That results in memory issues, among other things. 
2. Currently, if your tool would normally only have enough wear left to mine a block or two, you can still cut down a whole tree or generally mine more than the tool should allow. 

This needs to be fixed.

**Comment by drogars** [comment]
I agree, this is a pretty big deal.  I fixed it locally by removing several `node_name` sections from the `mineable_nodes.lua` file.  I removed dirt, stone, most of the metal blocks, and leaves.  This prevents the main issue you reported.  The original author already has a bug for your second item, though it looks like development on this as slowed a little bit.

**Comment by echosa** [comment]
Sorry, I've stopped playing Minetest for a while now. I've just been too busy. That means I also stopped working on mine_it_all. If someone else has bug fixes, please feel free to send in a PR.

**Comment by drogars** [comment]
If I get something together that looks good enough to submit a :pr: I will.
